platform-cloud/docs/studios/add-studio.md

```diff
@@ -2,82 +2,124 @@
 title: "Add a Studio"
 description: "Add a Studio in Platform."
 date created: "2025-09-04"
-last updated: "2025-10-21"
+last updated: "2025-09-04"
 tags: [data, session, studios]
 ---
 
 Select the **Studios** tab, and then select **Add Studio**.
 
 :::info[**Prerequisites**]
-You will need the following to get started:
-
-- Valid credentials for accessing cloud storage resources
-- **Maintain** role permissions (minimum)
-- A compute environment with sufficient resources (scale based on data volume):
-    - **CPUs**: 2 (minimum)
-    - **Memory**: 8192 MB  (minimum)
-- [Data Explorer](../data/data-explorer) enabled
+Before you get started, you need:
+
+- Valid credentials to access your cloud storage data resources.
+- At least the **Maintain** role set of permissions.
+- A compute environment with sufficient resources. This is highly dependent on the volume of data you wish to process, but at least 2 CPUs allocated with 8192 MB of memory is recommended.
+- [Data Explorer](../data/data-explorer) enabled.
```

Comment:
I think you can remove this now. Data Explorer is enabled by default and Enterprises have to explicitly turn it off.